### PR TITLE
AHTI-100 Add Image and License models and import data

### PR DESCRIPTION
**Pull request by charn**
This PR adds models for feature images and image licenses. Image data is imported from MyHelsinki places API.

### CODE REVIEW

File: features/models.py [hunk 1]
```diff
@@ -74,3 +74,42 @@ def __str__(self):
...
+        verbose_name = _("image")
+        verbose_name_plural = _("images")
+        ordering = ("id",)
+
+    def __str__(self):
+        return self.url
+
+
+class License(models.Model):
+    name = models.CharField(verbose_name=_("name"), max_length=200)
```
Comment (igordavydsson) [inline features/models.py]: just the name without the license text?

File: features/importers/myhelsinki_places/importer.py [hunk 2]
```diff
@@ -20,22 +20,42 @@ def import_features(self):
...
-                ),
-                "source_type": st,
-            }
-
-            Feature.objects.language("fi").update_or_create(
-                source_type=st, source_id=f"{place['id']}", defaults=values,
-            )
+            feature = self.import_feature(place, st)
+
+            for image in place["description"]["images"] or []:
```
Comment (igordavydsson) [inline features/importers/myhelsinki_places/importer.py]: is this `KeyError` safe?

File: utils/pytest.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import re
+
+
+class pytest_regex:
```
Comment (igordavydsson) [inline utils/pytest.py]: oh, neat util 👍 👍 👍

File: features/importers/myhelsinki_places/importer.py (hunk 2 above)
Comment (igordavydsson) [reply features/importers/myhelsinki_places/importer.py]: ah, nevermind 🙂

File: features/models.py (hunk 1 above)
Comment (charn) [reply features/models.py]: It's currently not 100% how the license text part will be handled. Current plan would be to have a url to the license, but I'll add the url field once it's clear where we can get the url. I guess it's also possible that we can (maybe in addition to the url) have a field for the license text. I made the field translatable as per the initial DB design.

Comment (igordavydsson) [reply features/models.py]: ok, well, it is a good solution for the MVP 👍

**Pull request merged**